### ISSUE TITLE
Parasite Eve 2 Pal Black Screen

### ISSUE THREAD

**Issue by naxeras** [DKWDRV/DKWDRV#19]
Parasite Eve 2 PAL version, black screen after MDEC video.

**Comment by DKWDRV** [comment]
Try 1.7.6
Confirm on your side.

**Comment by naxeras** [comment]
Boots now but freezes with black screen after the FMV intro.

**Comment by DKWDRV** [comment]
At least this version works fine for me.
http://redump.org/disc/118/

**Comment by naxeras** [comment]
> At least this version works fine for me. http://redump.org/disc/118/

The issue is with the spanish version of the game not the english version:

http://redump.org/disc/1122/
https://psxdatacenter.com/games/P/P/SLES-02561.html

**Comment by LeandroFerreira445** [comment]
> ### Parasite Eve 2 Disco 1 [SLES_025.58]:
> SCPH-90008 > Modchip Mars Pro GM-816HD desabilitado > OpenTuna > wLE_kHn > DKWDRV 1.7.6 USB > Congela nesta tela se você for para Pierce:
> 
> ![10341512-parasite-eve-ii-playstation-começando-o-jogo](https://private-user-images.githubusercontent.com/59209149/352311985-79e1cc53-6872-46d1-9ce2-1255775dc761.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxODYxNzQsIm5iZiI6MTcyMjE4NTg3NCwicGF0aCI6Ii81OTIwOTE0OS8zNTIzMTE5ODUtNzllMWNjNTMtNjg3Mi00NmQxLTljZTItMTI1NTc3NWRjNzYxLmpwZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI4VDE2NTc1NFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTNhN2JiMTNhZmMwOGVmMTlmMDVlNTBlOTA3MGJhNmI4MzhjZGEzMmE2NGYyY2UzODc5NTg0OGZlNGM5ZGIyOTYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.8hmUImXEXmmKRKCEc7JQ3xbE-8LgBZu8BL8sSdeIJc8)

Também tive o mesmo problema nessa parte

**Comment by DKWDRV** [comment]
It has been a while here that I have to go around with different super specific version and now I must also discover who Pierce is. So either be very specific on how I arrive to the "Pierce" and what is happening or else I will be forced to close this issue and let somebody else to make a new one with detailed info on what is happening if he really cares about the game.

Sorry, there is so much to debug that I simply don't have time to learn the games one by one, even if "Pierce" is super easy to find.

**Comment by naxeras** [comment]
> specific

@10522 and @LeandroFerreira445 are talking about other versions of the same game, but the original issue, is for the spanish version of the game. 
Freezes after the fmv. Im posted the redump link and psx database link and the game code, if you need more info please tell me about it.

If helps, pal english version of the game works after the fmv how you say it.

Regards

**Comment by LeandroFerreira445** [comment]
Parasite Eve II (Spain) (PAL)
ID: SLES_025.61
SCPH-90006
USB
DKWDRV-1.7.6

stuck here
![20240728_160742](https://github.com/user-attachments/assets/cf810cb4-38bd-4d81-9ef1-c780d03f9f87)

**Comment by naxeras** [comment]
> Parasite Eve II (Spain) (PAL) ID: SLES_025.61 SCPH-90006 USB DKWDRV-1.7.6
> 
> stuck here ![20240728_160742](https://private-user-images.githubusercontent.com/169464626/352861907-cf810cb4-38bd-4d81-9ef1-c780d03f9f87.jpg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjIxOTQ4NjQsIm5iZiI6MTcyMjE5NDU2NCwicGF0aCI6Ii8xNjk0NjQ2MjYvMzUyODYxOTA3LWNmODEwY2I0LTM4YmQtNGQ4MS05ZWYxLWM3ODBkMDNmOWY4Ny5qcGc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNzI4JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDcyOFQxOTIyNDRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04NWMyM2NjZDVkYTY5OTY0YjVlY2MwMDNjYjU3Y2NlNDA4NjRlNTk4ZWIxMjI3NzhiNTc2MjBhMmVlZjIzZmUwJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.led0ZfUjpw6xm-f31240cI1opXYovfPra5yRLB_2u4k)

Yes, I have the exact same issue with the same version of the game @DKWDRV Please, if you need anything else to fix the problem, ask for it.

Regards

**Comment by rykotxet** [comment]
I confirm this game (PAL/Spain) freezes after FMV intro, just like @LeandroFerreira445 said. Tested on SCPH-90004, unmodded, with DKWDRV v1.7.6, loading from USB.

**Comment by jackfloflores-afk** [comment]
Additional test results:

Console:
PS2 Slim SCPH-90001 (Deckard)

Loading method:
USB

DKWDRV versions tested:
- v1.7.3
- v1.7.6h
- v1.7.6i

PAL version:
Disc 1: SLES-02561
Disc 2: SLES-12561

Results:
- Original image: boots correctly.
- Paradox patched image: same behavior.
- Game is playable until leaving Douglas' gun shop.
- The screen turns black, but the background music keeps playing indefinitely.
- This happens every time.

NTSC-U version:
Disc 1: SLUS-01042
Disc 2: SLUS-01055

Results:
- Game boots.
- After pressing START on the title screen, the game freezes immediately.
- Same behavior on v1.7.3 and v1.7.6i.

The PAL version also requires SBI/LibCrypt support in emulators.

I can perform additional tests if needed.

**Comment by J013k** [comment]
I have the same issue:
> ### Parasite Eve 2 Disc 1 [SLES_025.58]:
> SCPH-90008 > Modchip Mars Pro GM-816HD disabled > OpenTuna > wLE_kHn > DKWDRV 1.7.6 USB > Freezes on this screen if you go to Pierce:
> 
> <img width="640" height="480" alt="Image" src="https://github.com/user-attachments/assets/fac9e056-d5bd-4a7f-9dd8-3d674e85be3b" />

Name: Parasite Eve II (Disc 1)
ID: [SLES_025.58](http://redump.org/disc/118/)
Region: EU
Console: SCPH-77004 with FMCB 1.966, without any modchip or MechaPwn 
Device: through USB
DKRDRV: 1_7_6b... 1_7_6m.

It always freezes when I go down:
<img width="640" height="480" alt="Image" src="https://github.com/user-attachments/assets/fac9e056-d5bd-4a7f-9dd8-3d674e85be3b" />
To be add this is the first playable moment in this game shortly after pressing new game.

**Comment by DKWDRV** [comment]
@naxeras @J013k @10522 @LeandroFerreira445 your reports helped a lot!  Try 1.7.6p, fixed

**Comment by J013k** [comment]
With version 1.7.6p the game gets stuck after starting new game on this screen:
<img width="656" height="97" alt="Image" src="https://github.com/user-attachments/assets/3632af22-e2fc-440b-bb0c-e7301faaab15" />

With a previous version 1.7.6o I was able to pass this screen and watch a short FMV.
